miso-web/src/main/webapp/scripts/library_hot.js

```diff
@@ -476,108 +493,112 @@ Library.hot = {
   /**
    * Gets tagBarcodes data from server
    */
-  getBarcodePositionsOnly: function (strat) {
-    jQuery.get("../../barcodePositionsJson", {strategy : strat},
-      function(posData) {
-        // set tagBarcodeData
-        Hot.dropdownRef.tagBarcodes[strat]['1'] = {};
-        if (posData['numApplicableBarcodes'] == 2) {
-          
-          jQuery.get('../../tagBarcodesJson', {strategy : strat , position: 2},
-            function (bc2Data) {
-              Library
-              Hot.dropdownRef.tagBarcodes[strat]['2'] = bc2Data.tagBarcodes;
+  getBarcodePositionsOnly: function (tbFamily) {
+    if (tbFamily) {
+      jQuery.get("../../barcodePositionsJson", {tagBarcodeFamily : tbFamily},
+        function(posData) {
+          // set tagBarcodeData
+          Hot.dropdownRef.tagBarcodes[tbFamily]['1'] = {};
+          if (posData['numApplicableBarcodes'] == 2) {
```

Comment:
Consider rolling this into a loop: `for (var pos = 1; pos <= posData['numApplicableBarcodes']; pos++)`